<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockBrowser</class>
 <widget class="QDialog" name="BlockBrowser">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
    background-color: rgb(32, 32, 32);</string>
  </property>
  <layout class="QGridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>21</number>
   </property>
   <property name="topMargin">
    <number>21</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_be" native="true">
     <widget class="QWidget" name="layoutWidget00">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="font">
          <font>
           <family>Open Sans,sans-serif</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20px; font-weight:600; color:#ffffff;&quot;&gt;Block Explorer &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget01">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>501</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
        background-color: rgba(20, 20, 20, 0.35);
       </string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <item row="18" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="1">
        <widget class="QPushButton" name="txButton">
         <property name="text">
          <string>Decode Transaction</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>Inputs:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="pawBox">
         <property name="text">
          <string>0000 KH/s</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="valueLabel">
         <property name="text">
          <string>Value out:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="txLabel">
         <property name="text">
          <string>Transaction ID:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="hardBox">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="txID">
         <property name="text">
          <string>000</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="inputBox">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="valueBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="hardLabel">
         <property name="text">
          <string>Block Difficulty:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="heightBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="heightLabel_2">
         <property name="text">
          <string>Block Height:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="timeBox">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="blockButton">
         <property name="text">
          <string>Jump to Block</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="merkleLabel">
         <property name="text">
          <string>Block Merkle:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="hashLabel">
         <property name="text">
          <string>Block Hash:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="feesBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="pawLabel">
         <property name="text">
          <string>Block Hashrate:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="feesLabel">
         <property name="text">
          <string>Fees:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="bitsLabel">
         <property name="text">
          <string>Block nBits:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="bitsBox">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="merkleBox">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0x0</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Outputs:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="outputBox">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="nonceLabel">
         <property name="text">
          <string>Block nNonce:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="nonceBox">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>Block Timestamp:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="hashBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0x0</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLineEdit" name="txBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string comment="Transaction id" extracomment="Transaction id"/>
         </property>
         <property name="placeholderText">
          <string comment="Transaction ID" extracomment="Transaction ID"/>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="gridFrame_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>580</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <property name="horizontalSpacing">
          <number>30</number>
         </property>
         <property name="verticalSpacing">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Block Height&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="heightBox_3">
           <property name="text">
            <string>00000</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;POW Block Reward&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="rewardBox_2">
           <property name="text">
            <string>00000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;POW difficulty&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="diffBox_2">
           <property name="text">
            <string>0.0002</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;POS difficulty&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="diffBox2_2">
           <property name="text">
            <string>0.0002</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Net Hashrate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="pawrateBox_2">
           <property name="text">
            <string>0 MH/s</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label3_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Last block time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="localBox_2">
           <property name="text">
            <string>0 KH/s</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label1_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Connections&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="connectionBox_2">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Total Volume&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="volumeBox_2">
           <property name="text">
            <string>000</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Total stake weight&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="minBox_2">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Network stake weight&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="maxBox_2">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mining Phase&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="cBox_2">
           <property name="text">
            <string>PoW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" rowspan="3">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>240</width>
            <height>389</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap>:/images/stat_logo</pixmap>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget02">
      <property name="geometry">
       <rect>
        <x>505</x>
        <y>0</y>
        <width>311</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Open Sans,sans-serif</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20px; font-weight:600; color:#ffffff;&quot;&gt;Espers Statistics&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>390</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:darkgrey;</string>
      </property>
      <property name="text">
       <string>Update Statistics</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
